### PR TITLE
Feature/cc 143 nav footer bar background seo

### PR DESCRIPTION
**Pull request by STY1997**
navBar & footer & SEO
Screen Shot 2022-11-29 at 08 50 39
Screen Shot 2022-11-29 at 08 51 37
Screen Shot 2022-11-29 at 08 51 48
Screen Shot 2022-11-29 at 08 50 52
Screen Shot 2022-11-29 at 08 51 21

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -2,17 +2,32 @@
...
+          400: "#3D3D3D",
+          500: "#323232",
+        },
+        typography: {
+          100: "#888888",
+          200: "#A5A5A5",
+          300: "#BDBDBD",
+          400: "#DBDBDB",
+          500: "#F9F9F9",
+        },
```
Comment (Cecilia0109) [inline tailwind.config.js:30]: Good job!

File: components/navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <nav className="bg-background-300 px-3 py-4 tablet:px-8 desktop:px-12 flex flex-row justify-between items-center">
+        {/* logo and logo text */}
+        <div className="flex flex-row gap-2 items-center">
+          <div className="relative w-8 h-8 tablet:w-9 tablet:h-9">
+            <Image src="/cc-logo.svg" alt="court canva logo" layout="fill" objectFit="contain" />
+          </div>
+          <p className="text-p tablet:text-logoText text-typography-500">CourtCanva</p>
+        </div>
+        {/* nav bar content */}
+        <ul className="hidden tablet:block text-p text-white w-[233px] h-6 tablet:flex tablet:justify-between tablet:items-center">
```
Comment (Cecilia0109) [inline components/navbar/index.tsx]: tablet:flex , tablet:block have same CSS properties can not apply at the same time

File: components/navbar/NavModal.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        </li>
+        <li>
+          <a href="#">Design</a>
+        </li>
+        <li>
+          <a href="#">About us</a>
+        </li>
+      </ul>
+    </nav>
+  );
```
Comment (Cecilia0109) [inline components/navbar/NavModal.tsx]: it is better to have a hover effect on the menu list.

Comment (tatoMa) [reply components/navbar/NavModal.tsx]: The hover effect UI has been added on the Figma file.

File: components/navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+const NavBar = ({ open }: Props) => {
+  return (
+    <>
+      <nav className="bg-background-300 px-3 py-4 tablet:px-8 desktop:px-12 flex flex-row justify-between items-center">
+        {/* logo and logo text */}
+        <div className="flex flex-row gap-2 items-center">
+          <div className="relative w-8 h-8 tablet:w-9 tablet:h-9">
+            <Image src="/cc-logo.svg" alt="court canva logo" layout="fill" objectFit="contain" />
+          </div>
+          <p className="text-p tablet:text-logoText text-typography-500">CourtCanva</p>
```
Comment (tatoMa) [inline components/navbar/index.tsx]: The font style is not as same as the design.

File: components/navbar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        {/* nav bar content */}
+        <ul className="hidden tablet:block text-p text-white w-[233px] h-6 tablet:flex tablet:justify-between tablet:items-center">
+          <li>
+            <a href="#">Home</a>
+          </li>
+          <li>
+            <a href="#">Design</a>
+          </li>
+          <li>
+            <a href="#">About us</a>
```
Comment (tatoMa) [inline components/navbar/index.tsx]: All these links can be jumped to the section with the same name of the id defined. eg <a href="#section-x"> and you have a section with id="section-x"

**Comment by tatoMa** [comment]
I don't know if it is only me. When close the mobile menu, the menu flashes after it closed.

File: tailwind.config.js [hunk 6]
```diff
@@ -90,6 +128,17 @@ module.exports = {
         introToRight2: "introToRight 0.5s cubic-bezier(0.175, 0.885, 0.320, 1.275) 0.5s both",
         introToLeft1: "introToLeft 0.5s cubic-bezier(0.175, 0.885, 0.320, 1.275)",
         introToLeft2: "introToLeft 0.5s cubic-bezier(0.175, 0.885, 0.320, 1.275) 0.5s both",
+        navModalFadeIn: "navModalFadeIn 0.3s ease-in-out",
+        navModalFadeOut: "navModalFadeOut 0.3s ease-in-out",
+      },
+      screens: {
+        tablet: "834px",
+        desktop: "1280px",
```
Comment (tatoMa) [inline tailwind.config.js]: Please use the default responsive breakpoint instead of customising them.

https://tailwindcss.com/docs/responsive-design

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**